libraries-testing/src/test/java/com/baeldung/archunit/smurfs/SmurfsArchUnitTest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * 
+ */
+package com.baeldung.archunit.smurfs;
+
+
+import com.tngtech.archunit.core.domain.JavaClasses;
+import com.tngtech.archunit.core.importer.ClassFileImporter;
+import com.tngtech.archunit.lang.ArchRule;
+import com.tngtech.archunit.library.Architectures.LayeredArchitecture;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.noClasses;
+import static com.tngtech.archunit.library.Architectures.layeredArchitecture;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author Philippe
+ *
+ */
+public class SmurfsArchUnitTest {
+    
+    @Test
+    public void givenPresentationLayerClasses_thenWrongCheckFails() {        
+        JavaClasses jc = new ClassFileImporter().importPackages("com.baeldung.archunit.smurfs");
+        
+        ArchRule r1 = classes()
```

Comment:
There are some issues with Fluent API formatting. The formatted recommended in the Author's documentation should help. Sometimes even after using the formatter, may still be some issues you have to tweak manually.